src/zinc/hal/stm32l1/usart.rs

```diff
@@ -131,16 +131,17 @@ impl CharIO for Usart {
   }
 }
 
-impl fmt::Writer for Usart {
-  fn write_str(&mut self, s: &str) -> fmt::Result {
-    use core::str::StrExt;
-    for b in s.bytes() {
-      wait_for!(self.reg.sr.transmit_data_empty());
-      self.reg.dr.set_data(b as u16);
-    }
-    Result::Ok(())
-  }
-}
+// TODO: encoding::types Writer
```

Comment:
I'll do if this one stays unfixed. That one was not supposed to be pushed though :-)